src/overmap.cpp

```diff
@@ -1092,13 +1093,23 @@ struct mutable_overmap_placement_rule {
     void decrement() {
         --max;
     }
+};
+
+struct mutable_overmap_placement_rule {
+    std::string overmap;
+    int_distribution max = int_distribution( INT_MAX );
+    int weight = INT_MAX;
+
+    mutable_overmap_placement_rule_remainder realise() const {
+        return { overmap, max.sample(), weight };
+    }
 
     void deserialize( JsonIn &jin ) {
```

Comment:
Note #51449, BTW.

Reply:
Cool; good to see that merged already.  I will update accordingly.